### ISSUE TITLE
Button on Hub to "download repo as zip"

### ISSUE THREAD

**Issue by xalteropsx** [huggingface/hub-docs#1965]
the problem with git clone

it download the file 4time size of original size and disk usage is also spammed
please consider provide an single download button on website itself

need to avoid uses of git cli

**Comment by louisbuchbinder** [comment]
yes +1 it would be great to download a repository via a url as a tar.gz or zip. Thank you!

**Comment by Wauplin** [comment]
moving to hub-docs as it's related to the HF Hub, not to the Python SDK
